### PR TITLE
reference.conf: Add SW360 scan results storage

### PR DESCRIPTION
**Pull request by tsteenbe**
See commit for details

Signed-off-by: Thomas Steenbergen <thomas.steenbergen@here.com>

### CODE REVIEW

**Review by sschuberth — APPROVED** [review]

**Review by bs-ondem — APPROVED** [review]
Looks good, but please fix https://github.com/oss-review-toolkit/ort/blob/4f2be4bdc96d6859057a3732704cfa4f8b9f45bb/model/src/test/kotlin/config/OrtConfigurationTest.kt#L71-L107 in order pass CI checks.

**Comment by tsteenbe** [comment]
@bs-ondem  Thanks for the hint, first time writing Kotlin tests, hope I did it right.

File: model/src/test/kotlin/config/OrtConfigurationTest.kt [hunk 1]
```diff
@@ -97,6 +97,16 @@ class OrtConfigurationTest : WordSpec({
                     val cdStorage = this["clearlyDefined"]
                     cdStorage.shouldBeInstanceOf<ClearlyDefinedStorageConfiguration>()
                     cdStorage.serverUrl shouldBe "https://api.clearlydefined.io"
+
```
Comment (mnonnenmacher) [inline model/src/test/kotlin/config/OrtConfigurationTest.kt:100]: Detekt complains about trailing spaces in this line.

File: model/src/test/kotlin/config/OrtConfigurationTest.kt [hunk 2]
```diff
@@ -69,7 +69,7 @@ class OrtConfigurationTest : WordSpec({
                 }
 
                 storages shouldNotBeNull {
-                    keys shouldContainExactlyInAnyOrder setOf("local", "http", "clearlyDefined", "postgres")
+                    keys shouldContainExactlyInAnyOrder setOf("local", "http", "clearlyDefined", "postgres", "sw360")
```
Comment (mnonnenmacher) [inline model/src/test/kotlin/config/OrtConfigurationTest.kt:72]: It's "sw360Configuration" instead of "sw360".

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

**Review by fviernau — APPROVED** [review]

File: model/src/test/kotlin/config/OrtConfigurationTest.kt (hunk 2 above)
Comment (tsteenbe) [reply model/src/test/kotlin/config/OrtConfigurationTest.kt]: Fixed

File: model/src/test/kotlin/config/OrtConfigurationTest.kt (hunk 1 above)
Comment (tsteenbe) [reply model/src/test/kotlin/config/OrtConfigurationTest.kt]: Fixed

**Review by bs-ondem — APPROVED** [review]

**Review by mnonnenmacher — APPROVED** [review]

**Pull request merged**